### DISCUSSION TITLE
stat-computing-r-python/part-wrangling/07-datetime

### DISCUSSION THREAD

**Discussion by giscus** [srvanderplas/stat-computing-r-python#11] [General]
# stat-computing-r-python/part-wrangling/07-datetime

https://srvanderplas.github.io/stat-computing-r-python/part-wrangling/07-datetime.html

**Comment by rlnayituriki** [comment]
The first months of feeds link does not work.

**Reply by srvanderplas** [reply]
Should be fixed within the next 30 mins as the book recompiles.
